cmake_scripts/default_build_setup.cmake

```diff
@@ -28,7 +28,7 @@ option(DISABLE_CXX11_ABI
 
 if (UNIX)
     # flags apply for both Linux and OSX!
-    set(CMAKE_CXX_FLAGS "-Wall -Wextra")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra $")
```

Comment:
@mariusvniekerk Is the trailing `$` here intended? I guess this breaks CI.

Reply:
nope lemme fix